Assets/Plugins/PlayroomPlugin.jslib

```diff
@@ -23,6 +23,16 @@ mergeInto(LibraryManager.library, {
       });
     }
 
+    function OnLaunchCallBack() {
```

Comment:
Did you test this @momintlh ?

Reply:
Yes, I did, its working fine @SaadBazaz.

Reply:
Looks good! Resolving.